src/views/directMessages/containers/index.js

```diff
@@ -120,6 +127,9 @@ class DirectMessages extends React.Component<Props, State> {
       data.user.directMessageThreadsConnection.pageInfo &&
       data.user.directMessageThreadsConnection.pageInfo.hasNextPage;
 
+    const isInActiveMessages = this.props.match.path === '/messages';
+    const isInArchivedMessages = this.props.match.path === '/messages/archived';
```

Comment:
you have couple of places where you are using those strings, better to have a consts for them ;)

Reply:
Yeah, generally speaking we have hardcodes routes all over the place. I guess it could be a good start for another PR, maybe a `good first pr` type of work.

Reply:
Not only routes but strings in general. Moving forward it would be great to move Errors/Messages and routes somewhere else :-)